test/integration/consul-container/libs/service/connect.go

```diff
@@ -142,18 +142,24 @@ func (g ConnectContainer) GetStatus() (string, error) {
 	return state.Status, err
 }
 
+type SidecarConfig struct {
+	SidecarServiceName string
```

Comment:
Given the name of the struct, I would go with just `ServiceName` here to avoid stutter

Reply:
I was on the fence, I didn't want this to be confused for the name of the service that this is a sidecar for, maybe just `Name`?

Reply:
I don't have a very strong opinion here, just thought I'd call it out while also approving if you wanted to leave as-is